pkg/printers/internalversion/describe_test.go

```diff
@@ -2133,14 +2133,14 @@ URL:	http://localhost
 				"name":              "MyName",
 				"namespace":         "MyNamespace",
 				"creationTimestamp": "2017-04-01T00:00:00Z",
-				"resourceVersion":   123,
+				"resourceVersion":   int64(123),
```

Comment:
this is unrelated. Can you move it to another PR? (I cannot approve this).

Reply:
Most likely it was causing a failure because `123` literal has an `int` type by default but deep copy only likes `float64` and `int64`. But this fix is no longer needed because we are not making a deep copy.